src/tutorial/runtime/introduction.rst

```diff
@@ -291,65 +337,165 @@ Where Do All The Files Go?
+   will install your workflow in ``~/cylc-run/my_workflow/runN``.
 
-   Next we can run the workflow using the ``cylc play`` command.
+From a workflow developed in an arbitrary location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: sub
 
-   cylc play <name>
+   cylc install -C /some/location/my_other_workflow
 
 .. ifnotslides::
 
-   The ``name`` is the name of the :term:`run directory` (i.e. ``<name>``
-   would be ``foo`` in the above example).
+   will install your workflow in ``~/cylc-run/my_other_workflow/runN``.
 
-.. note::
+   .. note::
+
+      You can still develop workflows in ``~/cylc-run``, but this is not
+      recommended because development work may
+      change the behaviour of the workflow as it is running.
+
+Validating A Workflow
+---------------------
+
+.. ifnotslides::
 
-   In this tutorial we are writing our workflows in the ``cylc-run`` directory.
+   It is a good idea to check a workflow definition for errors before running
+   it. Cylc provides a command which automatically checks the validity of
+   workflow configurations - ``cylc validate``:
+
+.. code-block:: console
+
+   $ cylc validate <path/to/workflow>
+   $ cylc validate <workflow_id>  # For workflow in `~/cylc-run/`
+
+.. ifnotslides::
+
+   Here ``<path/to/workflow>`` is the path to the workflow's location within the
+   root source directory (e.g. ``~/cylc-src/``).
+
+
+Running a workflow
+------------------
+
+.. ifnotslides::
+
+   Now we have installed and validated our workflow we can run the
+   workflow using the ``cylc play`` command.
+
+.. code-block:: console
+
+   $ cylc play <workflow_id>
+
+.. ifnotslides::
+
+   The :term:`workflow id` is the path of the :term:`run directory` relative
+   to ``~/cylc-run``.
 
-   However, you should write real workflows in a separate source location
-   (Cylc expects ``~/cylc-src`` by default) and then use ``cylc install`` to
-   install them to the run directory before use. For more information, see
-   :ref:`Installing-workflows`.
 
 Generated Workflow Files
 ------------------------
 
+Numbered run directories
+^^^^^^^^^^^^^^^^^^^^^^^^
+
 .. ifnotslides::
 
-   Cylc generates files and directories when it runs a workflow, namely:
+   .. seealso::
+
+      :ref:`Installing-workflows` for a fuller description of Cylc install,
+      including the option of naming rather than numbering runs.
+
+   By default ``cylc install`` will install your workflow in a new
+   numbered run directory each time you run ``cylc install``:
+
+.. code-block:: console
+
+   $ cylc install my_workflow
+   INSTALLED my_workflow/run1 from ...
+   $ cylc install my_workflow
+   INSTALLED my_workflow/run2 from ...
+
+   # The most recent ``runX`` directory is symlinked to ``runN``
+   $ ls -l ~/cylc-run/my_workflow/runN
+   ~/cylc-run/baz/runN -> run2
+
+.. ifnotslides::
+
+   Each time you run ``cylc install``, a symlink called ``runN`` is
+   created/updated, pointing to the newest run directory.
```

Comment:
Duplicate of `The most recent runX directory is symlinked to runN`.

Should try to reduce this (note the tutorial isn't supposed to be a substitute to the user guide, we need to cover topics quickly and can't spend too long dwelling on the CLI)

```suggestion
```